tests/python-pytest/onnx/test_operators.py

```diff
@@ -923,3 +923,16 @@ def test_onnx_export_convolution(tmp_path, dtype, shape, num_filter, num_group,
                   **kwargs)
     inputs = [x, w] if no_bias else [x, w, b]
     op_export_test('convolution', M, inputs, tmp_path)
+
+
+@pytest.mark.parametrize('dtype', ['float16', 'float32'])
+@pytest.mark.parametrize('num_outputs', [1, 3, 9])
+@pytest.mark.parametrize('axis', [1, 2, -1, -2])
+@pytest.mark.parametrize('squeeze_axis', [True, False])
```

Comment:
Can you provide a numerical value to squeeze_axis? Does the onnx export function correctly see 'True' when passed a 1?

Reply:
I added '1' as an equivalent to 'True', but I think technically users can also use other values such as 4, 123, although i doubt they would actually use it that way